application/deed/views.py

```diff
@@ -129,3 +132,50 @@ def sign_deed(deed_reference):
 @deed_bp.route('/<deed_reference>/make-effective', methods=['POST'])
 def make_effective(deed_reference):
     return status.HTTP_200_OK
+
+
+@deed_bp.route('/<deed_reference>/request-auth-code', methods=['POST'])
+def send_sms(deed_reference):
+    request_json = request.get_json()
+    borrower_token = request_json['borrower_token']
+    if borrower_token is not None and borrower_token != '':
+        borrower = Borrower.get_by_token(borrower_token.strip())
+
+        if borrower is not None:
+            borrower_phone_number = borrower.phonenumber
+
+            code = generate_sms_code(deed_reference, borrower_token)
+
+            message = code + " is your digital mortgage authentication code."
+            twilio_phone_number = config.TWILIO_PHONE_NUMBER
```

Comment:
my preference would be to assign the constant directly into the literal below, as the variable is otherwise not used.